EvidenceApi/V1/UseCase/FindDocumentSubmissionsByResidentIdUseCase.cs

```diff
@@ -35,36 +35,40 @@ public async Task<List<DocumentSubmissionResponse>> ExecuteAsync(DocumentSubmiss
                 ServiceRequestedBy = request.ServiceRequestedBy,
                 ResidentId = request.ResidentId
             };
-            var found = _evidenceGateway.GetEvidenceRequests(evidenceRequestSearchQuery);
-
-            var documentSubmissions = new List<DocumentSubmission>();
-
-            foreach (var er in found)
-            {
-                documentSubmissions = documentSubmissions.Concat(_evidenceGateway.FindDocumentSubmissionByEvidenceRequestId(er.Id)).ToList();
-            }
+            var evidenceRequests = _evidenceGateway.GetEvidenceRequests(evidenceRequestSearchQuery);
 
             var result = new List<DocumentSubmissionResponse>();
 
-            foreach (var ds in documentSubmissions)
+            foreach (var evidenceReq in evidenceRequests)
             {
-                var documentType = FindDocumentType(ds.DocumentTypeId);
-                var claim = await _documentsApiGateway.GetClaimById(ds.ClaimId).ConfigureAwait(true);
-                if (claim.Document == null)
+                var documentSubmissions = _evidenceGateway.FindDocumentSubmissionByEvidenceRequestId(evidenceReq.Id);
```

Comment:
```suggestion
                var documentSubmissions = _evidenceGateway.FindDocumentSubmissionsByEvidenceRequestId(evidenceReq.Id);
```
I think the name of that function needs to be changed (to use plural) now that we can have more document submissions for each evidence request

Reply:
Makes sense, addressed